crates/client/rpc/src/lib.rs

```diff
@@ -649,6 +676,16 @@ where
         let chain_id = self.chain_id()?;
         let chain_id = Felt252Wrapper(chain_id.0);
 
+        let transaction_hashes = self.get_cached_transaction_hashes(block.header().hash::<H>().into());
+        let mut transactions = Vec::with_capacity(block.transactions().len());
+        for (index, tx) in block.transactions().iter().enumerate() {
+            let hash = transaction_hashes
+                .as_ref()
+                .map(|vec| h256_to_felt(*vec.get(index).ok_or(StarknetRpcApiError::InternalServerError)?))
```

Comment:
avoid calling a Vec variable "vec".
I'm fine with `v` but something more expressive like `tx_hashes` is better